gobblin-core/src/main/java/gobblin/source/extractor/extract/restapi/RestApiExtractor.java

```diff
@@ -73,6 +73,27 @@ public RestApiExtractor(WorkUnitState state) {
 
   protected abstract RestApiConnector getConnector(WorkUnitState state);
 
+  protected void buildQuery(String inputQuery, String entity, JsonArray columnArray) {
```

Comment:
`columnArray` is not used in the function. Remove? I guess what you want is actually `List<String> columnList`.